reactapp/package.json

```diff
@@ -36,10 +36,10 @@
     "eject": "react-app-rewired eject",
     "precommit": "lint-staged",
     "app:start": "react-app-rewired start",
-    "prestart": "cd ../../../../../vendor/hyva-themes/magento2-hyva-checkout/src/view/frontend/web/css/tailwind/ && npm run build-dev && cp ../styles.css ../../../../../../../../../app/code/Hyva/CustomizeCheckout/reactapp/public/styles.css",
-    "prebuild": "cd ../../../../../vendor/hyva-themes/magento2-hyva-checkout/src/view/frontend/web/css/tailwind/ && npm run build-prod",
-    "tailwind-install": "cd ../../../../../vendor/hyva-themes/magento2-hyva-checkout/src/view/frontend/web/css/tailwind/ && npm install && cd ../../../../../../../../../app/code/Hyva/CustomizeCheckout/reactapp/",
-    "tailwind-update": "cd ../../../../../vendor/hyva-themes/magento2-hyva-checkout/src/view/frontend/web/css/tailwind/ && npm update && cd ../../../../../../../../../app/code/Hyva/CustomizeCheckout/reactapp/"
+    "prestart": "cd ../view/frontend/web/css/tailwind/ && npm run build-dev",
```

Comment:
Theoretically this can be replaced with `npm --prefix ../view/frontend/web/css/tailwind/ run build-dev`

The cd commands are not really needed.
See also https://stackoverflow.com/a/14867050/753676 and https://remarkablemark.org/blog/2020/10/23/npm-run-script-different-directory/

Reply:
Same for the others with `cd ../../...`.

Reply:
Thanks for the tip. Let me try that out 😉

Reply:
This is also updated 👍